lib/src/airship_config.dart

```diff
@@ -2,7 +2,7 @@ class AirshipConfig {
 
   final ConfigEnvironment? defaultEnvironment;
 
-  AirshipConfig({this.defaultEnvironment});
+  AirshipConfig(this.defaultEnvironment);
```

Comment:
This class needs to be extended to match the JSON that we have for the proxy. Take a look at react native for the types